app/services/proofing/lexis_nexis/verification_error_parser.rb

```diff
@@ -38,19 +38,38 @@ def verification_status
       attr_reader :base_error_message, :product_error_messages
 
       def parse_base_error_message
-        error_code = body.dig('Status', 'TransactionReasonCode', 'Code')
-        conversation_id = body.dig('Status', 'ConversationId')
-        reference = body.dig('Status', 'Reference')
-        tracking_ids = "(LN ConversationId: #{conversation_id}; Reference: #{reference}) "
+        return "Invalid status in response body: '#{verification_status}'" if !valid_status?
+
+        error_code = transaction_reason_code
+        if verification_status == 'error'
+          error_information = body.fetch('Information', {}).to_json
+          "Response error with code '#{error_code}': #{error_information}"
+        elsif error_code.nil?
+          'Verification failed without a reason code'
+        else
+          "Verification failed with code: '#{error_code}'"
+        end
+      end
 
-        return "#{tracking_ids} Verification failed without a reason code" if error_code.nil?
+      def transaction_reason_code
+        body.dig('Status', 'TransactionReasonCode', 'Code')
+      end
+
+      def conversation_id
+        body.dig('Status', 'ConversationId')
```

Comment:
code coverage seems to think this isn't covered?

Reply:
> code coverage seems to think this isn't covered?

Ah, it's unused now. Prior to 61b7fdef72c1903023d89437d8cce626a3240960 we were embedding the conversation ID in the error message.

Reply:
Should we remove it? Or do we still use it in success cases?

Reply:
I'll remove. The conversation ID is still used, but a step up in the response class, which is relevant to the intended change here since the goal is to make sure that we include the conversation ID even in these instances where previously we'd have raised errors on unexpected status codes.

https://github.com/18F/identity-idp/blob/2f7c0ee15cdca484d762de6c9d4a76bef7c1f3aa/app/services/proofing/lexis_nexis/response.rb#L36-L38